app/decorators/controllers/solidus_bolt/spree_checkout_controller/add_addresses_to_bolt.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module SolidusBolt
+  module SpreeCheckoutController
+    module AddAddressesToBolt
+      def before_delivery
+        spree_current_user&.addresses&.each do |address|
+          SolidusBolt::Accounts::AddAddressService.call(
```

Comment:
I don't think we just have to add the address here. Maybe it would be better if we make addresses in sync between solidus and bolt. In order to do it, we should scan the existing address and add it to the bolt account, only if the address is not already existing.

Reply:
added as new separate commit in https://github.com/nebulab/solidus_bolt/pull/96/commits/b4fbce51a9f1d78aa8bd5f00f65db70da7eaa0a4